typescript/infra/scripts/rebalancer/deploy-rebalancer.ts

```diff
@@ -0,0 +1,88 @@
+import { input } from '@inquirer/prompts';
+import path from 'path';
+
+import {
+  LogFormat,
+  LogLevel,
+  configureRootLogger,
+  rootLogger,
+} from '@hyperlane-xyz/utils';
+
+import { DeployEnvironment } from '../../src/config/environment.js';
+import { RebalancerHelmManager } from '../../src/rebalancer/helm.js';
+import { HelmCommand } from '../../src/utils/helm.js';
+import { validateRegistryCommit } from '../../utils.js';
+import {
+  assertCorrectKubeContext,
+  getArgs,
+  getWarpRouteIdsInteractive,
+  withMetrics,
+  withWarpRouteId,
+} from '../agent-utils.js';
+import { getEnvironmentConfig } from '../core-utils.js';
+
+function getRebalancerConfigPathPrefix(environment: DeployEnvironment) {
+  return `config/environments/${environment}/rebalancer/rebalancerConfigs`;
+}
+
+async function main() {
+  configureRootLogger(LogFormat.Pretty, LogLevel.Info);
+  const { environment, warpRouteId, metrics } = await withMetrics(
+    withWarpRouteId(getArgs()),
+  ).parse();
+
+  let warpRouteIds;
+  if (warpRouteId) {
+    warpRouteIds = [warpRouteId];
+  } else {
+    warpRouteIds = await getWarpRouteIdsInteractive();
+  }
+
+  const registryCommit = await input({
+    message:
+      'Enter the registry version to use (can be a commit, branch or tag):',
+  });
+  await validateRegistryCommit(registryCommit);
+
+  await assertCorrectKubeContext(getEnvironmentConfig(environment));
+
+  rootLogger.info(
+    `Deploying Rebalancer for Route ID: ${warpRouteIds.join(', ')}`,
```

Comment:
maybe maybe it bullet points instead, i imagine the line will get quite long given route ID length and the number of routes that could get added